canonical-web-and-design/vanilla-framework
# Updated watch:scss:skip-standalone - WIP

## Done

- Updated watch:scss:skip-standalone to build form build.scss

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/3941

## QA

- Open [demo](insert-demo-url)
- Check that the watch:scss:skip-standalone works

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
